### ISSUE TITLE
Sublime Text 3

### ISSUE THREAD

**Issue by gjaldon** [jugyo/SublimeRubyEval#5]
This doesn't seem to work for ST3. Any plans on updating it for ST3?

**Comment by jugyo** [comment]
I hope that. Please show me the error on ST3 if you could.

**Comment by fuadsaud** [comment]
Got this:

```
Traceback (most recent call last):
  File "/Applications/Sublime Text.app/Contents/MacOS/sublime_plugin.py", line 549, in run_
    return self.run(edit)
  File "RubyEval in /Users/fuad/Library/Application Support/Sublime Text 3/Installed Packages/RubyEval.sublime-package", line 58, in run
  File "RubyEval in /Users/fuad/Library/Application Support/Sublime Text 3/Installed Packages/RubyEval.sublime-package", line 49, in eval_as_ruby
NameError: global name 'unicode' is not defined
```

**Comment by jugyo** [comment]
Finally this has been fixed! 8399ffca2b0191853a647db9686867cddb708be4
